workers/analyzer/docker/Analyzer.Dockerfile

```diff
@@ -22,6 +22,7 @@
 # License-Filename: LICENSE
 
 ARG ANDROID_CMD_VERSION=11076708
+ARG BAZEL_VERSION=7.0.1
```

Comment:
Why not the latest 7.2.1? If there's a reason, please provide it in the commit message.

Reply:
This is just aligned with ORT :shrug: 

What do you think we should do?

Reply:
Hmm, I just checked, and running ORT core tests with Bazel 7.2.1 requires changes to test assets, so I'd propose to stick both to Bazel 7.0.1 for now, but mention the alignment in the commit message.